File: multimediaBack-end/multimedia/src/main/java/ao/isptec/multimedia/dto/AlbumConjuntoArtistas.java
package ao.isptec.multimedia.dto;

import java.util.List;

import ao.isptec.multimedia.model.Album;
import ao.isptec.multimedia.model.AlbumArtista;
import ao.isptec.multimedia.model.Artista;
import ao.isptec.multimedia.model.Musica;
import ao.isptec.multimedia.model.MusicaArtista;

public class AlbumConjuntoArtistas {

    private Album album;
    private List<AlbumArtista> conjuntoArtistas;

    public AlbumConjuntoArtistas(Album album, List<AlbumArtista> conjuntoArtistas) {
        this.album = album;
        this.conjuntoArtistas = conjuntoArtistas;
    }

    public Album getAlbum() {
        return album;
    }

    public void setAlbum(Album album) {
        this.album = album;
    }

    public List<AlbumArtista> getConjuntoArtistas() {
        return conjuntoArtistas;
    }

    public void setConjuntoArtistas(List<AlbumArtista> conjuntoArtistas) {
        this.conjuntoArtistas = conjuntoArtistas;
    }

}

File: isp-media-angular/src/app/models/AlbumConjuntoArtistas.ts
import { Album } from "./Album";
import { AlbumArtista } from "./AlbumArtista";
import { Artista } from "./Artista";
import { ConteudoGrupo } from "./ConteudoGrupo";
import { Grupo } from "./Grupo";
import { MembroGrupo } from "./MembroGrupo";
import { Musica } from "./Musica";
import { Utilizador } from "./Utilizador";

export class AlbumConjuntoArtistas {
    constructor(
        public album: Album,
        public conjuntoArtistas: Array<AlbumArtista>
    ) { }
}